src/ucp/wireup/wireup.h

```diff
@@ -155,23 +160,33 @@ ucs_status_t ucp_wireup_resolve_proxy_lanes(ucp_ep_h ep);
 void ucp_wireup_remote_connected(ucp_ep_h ep);
 
 /**
- * Check if TL has to be connected point to pont.
+ * Check if TL supports point to pont connections.
  *
  * @param [in]  worker       UCP worker.
  * @param [in]  rsc_index    resource index.
- * @param [in]  prefer_p2p   hint to prefer @ref UCT_IFACE_FLAG_CONNECT_TO_EP if
- *                           TL supports @ref UCT_IFACE_FLAG_CONNECT_TO_IFACE as
- *                           well.
  *
- * @return 1 if TL has to be connected point to pont, otherwise 0.
+ * @return 1 if TL supports point to pont connections, otherwise 0.
  */
 static inline int ucp_worker_is_tl_p2p(ucp_worker_h worker,
-                                       ucp_rsc_index_t rsc_index,
-                                       int prefer_p2p)
+                                       ucp_rsc_index_t rsc_index)
 {
     return ucp_worker_iface_is_tl_p2p(ucp_worker_iface_get_attr(worker,
-                                                                rsc_index),
-                                      prefer_p2p);
+                                                                rsc_index));
+}
+
+/**
+ * Check if TL supports connection to interface.
+ *
+ * @param [in]  worker       UCP worker.
+ * @param [in]  rsc_index    resource index.
+ *
+ * @return 1 if TL supports connection to interface, otherwise 0.
+ */
+static inline int ucp_worker_is_tl_2iface(ucp_worker_h worker,
+                                          ucp_rsc_index_t rsc_index)
+{
+    return ucp_worker_iface_is_tl_2iface(ucp_worker_iface_get_attr(worker,
```

Comment:
is ucp_worker_iface_is_tl_2iface ever used directly and not from ucp_worker_is_tl_2iface()?
if not need to unite them